tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -47,13 +48,25 @@ internal class LogFormatter
             _source = string.IsNullOrEmpty(settings.Source) ? null : settings.Source;
             _service = string.IsNullOrEmpty(serviceName) ? null : serviceName;
             _host = string.IsNullOrEmpty(settings.Host) ? null : settings.Host;
-            _tags = string.IsNullOrEmpty(settings.GlobalTags) ? null : settings.GlobalTags;
+            _tags = EnrichTagsWithAasMetadata(settings.GlobalTags, aasSettings);
             _env = string.IsNullOrEmpty(env) ? null : env;
             _version = string.IsNullOrEmpty(version) ? null : version;
         }
 
         internal delegate LogPropertyRenderingDetails FormatDelegate<T>(JsonTextWriter writer, in T state);
 
+        private string? EnrichTagsWithAasMetadata(string globalTags, ImmutableAzureAppServiceSettings? aasSettings)
+        {
+            if (aasSettings is null)
+            {
+                return string.IsNullOrEmpty(globalTags) ? null : globalTags;
+            }
+
+            var aasTags = $"{Tags.AzureAppServicesResourceId}:{aasSettings.ResourceId}";
+
+            return string.IsNullOrEmpty(globalTags) ? aasTags : aasTags + "," + globalTags;
```

Comment:
A const for what? We already have a const for the tag? 🤔

Reply:
For the separator, the comma

Reply:
oh, ok 😅

Reply:
Don't hate me for this, but I was mainly suggesting to share the consts with [this function](https://github.com/DataDog/dd-trace-dotnet/blob/208bc0ab1c0e8261dbcef136e6a27a8b03917725/tracer/src/Datadog.Trace/Logging/DirectSubmission/ImmutableDirectLogSubmissionSettings.cs#L195)/ My only worry was not to have a string but to change it in one place one day, and not in the other (even though unlikely to happen)

Reply:
> Don't hate me for this

:stare:

> but I was mainly suggesting to share the consts with [this function](https://github.com/DataDog/dd-trace-dotnet/blob/208bc0ab1c0e8261dbcef136e6a27a8b03917725/tracer/src/Datadog.Trace/Logging/DirectSubmission/ImmutableDirectLogSubmissionSettings.cs#L195)

We currently concatenate tags in a _lot_ of places. My personal inclination is to refactor all those consts out in a separate PR in that case, otherwise I'm going to be touching a lot of unrelated files

Reply:
ok that was a nitpicking anyway :)